### ISSUE TITLE
Copy of Numerical Forms Keypad Input/ Calendar Date Selection - Supporting Partner Accessibility Fixes - Level A- Keyboard

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#3088]
**User Story**
**As a Supporting Partner who relies on assistive technology, I need: 
- numerical form fields to allow keypad input. so that I can. use keypad for numerical entries 
- Date of Birth calendar date selection accessible so that I can navigate this form field entirely 

**Context:**
- Theses  issues were uncovered through accessibility audit via CITZ. 
- Detailed Spreadsheet for Student s inserted inline and also attached here: [2024_Supporting Information_Audit.xlsx](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN1RsQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--401fd26be3124e38981234d207ea635590b5dcd2/2024_Supporting%20Information_Audit.xlsx)
- All functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints.

**The Problem**
- Numerical form fields do not allow keypad input.
- Date of Birth calendar date selection inaccessible. Users relying on assistive technology may get trapped in the calendar function or be unable to navigate this form field entirely due to requiring the use of the calendar function. Also does not meet 2.1.2 No Keyboard Trap.

**Acceptance Criteria**
- [ ] Enable the use of keypad for numerical entries at 'Search for application'
- [ ] Make typing input available
- [ ] Content Provide Clear format instructions (devs can you suggest instruction?) 

<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMS9uQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a254a9b37eaba9d413a94fe8053c3d114cf272d0/image.png" alt="image.png" />

**Comment by michesmith** [comment]
How can we get these more succinct?
